### ISSUE TITLE
pytest order being set to alphabetical with reportportal configure pytest.ini

### ISSUE THREAD

**Issue by skinny-b** [reportportal/agent-python-pytest#130]
I noticed when trying to run some automation after having installed and setup pytest.ini to be capable of publishing to our reportportal server that what I was calling was no longer in the order I wanted.

```
(venv) bash-4.2$ uname -a
Linux ##### 3.10.0-957.27.2.el7.x86_64 #1 SMP Mon Jul 29 17:46:05 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
(venv) bash-4.2$ pip list
Package             Version
------------------- ---------
..
dill                0.3.1.1
..
pytest              5.2.0
pytest-reportportal 1.0.8
reportportal-client 3.2.3
..
(venv) bash-4.2$ pytest -vv tests/b.py tests/a.py
======================================================= test session starts ========================================================
platform linux -- Python 3.6.6, pytest-5.2.0, py-1.8.0, pluggy-0.13.0 -- #########
cachedir: .pytest_cache
rootdir: #####, inifile: pytest.ini
plugins: reportportal-1.0.8
collected 2 items

tests/a.py::test_run_one PASSED                                                                                              [ 50%]
tests/b.py::test_run_two PASSED                                                                                              [100%]

======================================================== 2 passed in 0.03s =========================================================
(venv) bash-4.2$ cat pytest.ini
[pytest]
rp_uuid = ####
rp_project = ####
rp_endpoint = #######
rp_verify_ssl = ####
Removing the configuration for reportportal via pytest.ini
(venv) bash-4.2$ rm pytest.ini
(venv) bash-4.2$ pytest -vv tests/b.py tests/a.py
======================================================= test session starts ========================================================
platform linux -- Python 3.6.6, pytest-5.2.0, py-1.8.0, pluggy-0.13.0 -- ############
cachedir: .pytest_cache
rootdir: ########
plugins: reportportal-1.0.8
collected 2 items

tests/b.py::test_run_two PASSED                                                                                              [ 50%]
tests/a.py::test_run_one PASSED                                                                                              [100%]

======================================================== 2 passed in 0.05s =========================================================
```

Code for (a|b).py consists of pass to illustrate this if it matters.
The sorting is enforced in plugin.py:331 under pytest_collection_modifyitems

**Comment by iivanou** [comment]
On one hand, it's bad practice to write tests that are dependent on each other. 

On the other hand, the changes made in PR #62  break Pytest's default collection behavior if test modules are specified in the strict order. This change is supposed to add support for nested hierarchy test items. But in fact, the sorting applied to test items does not make any influence on their appearance at the web interface. I'd like to see the real use cases for that sorting approach where that change takes effect. @s0undt3ch, could you please provide those?

**Comment by s0undt3ch** [comment]
Unfortunatly our PoC using RP which led me to update this plugin had been stalled and I don't know when we'll ever pick it up.
It was a long time ago and I don't recall the why for this change anymore, sorry.

**Comment by iivanou** [comment]
@skinny-b, Can you please comment out the following line in the plugin.py and confirm that your tests still report results correctly and in the appropriate order:

`items.sort(key=lambda f: (f.fspath, f.parent.name)`

**Comment by skinny-b** [comment]
From what I can tell yes, that works fine. Reportportal shows them in order and looks proper there.

**Comment by iivanou** [comment]
Fixed in #132.
